woocommerce/woocommerce-android
# Feature/9557 update store creation wording

Closes: #9557 


### Description

Please don't panic 😄 . 
While there are many file changes on the re PR most of them are super small UI changes or simply adding and removing drawable assets.

The biggest relevant changes were applied to `StoreCreationSummaryScreen` as that required a complete UI revamp. 

### Testing instructions
Using Figma designs for reference: 8sNT07kW3VEdnor4baI6ft-fi-387%3A40489
- Check login prologue matches the new wording and button texts shown in Figma
- Kick start store creation flow and check profiler steps match the exact wording as shown in Figma. I also removed the Help button from profiler steps.  
- Kick start store creation flow and check store summary screen matches all the new design from Figma:

<img width="300"  src="https://github.com/woocommerce/woocommerce-android/assets/2663464/78f648e6-2053-4fb6-8063-e3080d42f44d">
